repositories/src/main/scala/cromwell/pipeline/datastorage/formatters/package.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.pipeline.datastorage
+
+import play.api.libs.json.{ Format, JsError, JsResult, JsSuccess, JsValue, Json }
+import cats.Show
+import cromwell.pipeline.model.validator.Wrapped
+import cromwell.pipeline.model.wrapper.{ Name, Password, UserEmail, UserId, VersionValue }
+
+import scala.language.implicitConversions
+import cats.data.Validated.{ Invalid, Valid }
+import cats.implicits._
+import cromwell.pipeline.model.validator.cromwell.pipeline.model.validator.WrappedFactoryPattern
+
+package object formatters {
+  private def wrapperFormat[T, E, W <: Wrapped[T]](wrapperFactory: WrappedFactoryPattern[T, E, W])(
+    implicit show: Show[E],
+    format: Format[T]
+  ): Format[W] =
+    new Format[W] {
+      override def reads(json: JsValue): JsResult[W] =
+        wrapperFactory.validate(json.as[T]).map(wrapperFactory.create) match {
```

Comment:
Use `json.validate` instead of `json.as`, since it doesn't throw an exception. Also, you must reuse `.from` instead of `.validate` + `.create`

Reply:
"Use json.validate instead of json.as" I think, its a high time to make a list of Akka, Play good practices, on GitHub comments materials. I feel I need it

done